pkg/esxi/testdata/bootstate_twice.cfg

```diff
@@ -0,0 +1,4 @@
+kernel=b.b00
+kernelopt=zee
+bootstate=4
+bootstate=
```

Comment:
Wow, this is valid? Interesting

Reply:
from my reading of their code, it *seemed* like it. probably no harm in allowing it, whether they do it or not.